README.md

```diff
@@ -31,7 +31,8 @@ If you have satisfied the requirements from the Pre-Installation section, you ca
   * [Examples](#examples)
 * [Tests](#tests)
   * [Preparing Node](#preparing-node)
-  * [Running Tests](#running-tests)
```

Comment:
Please add some description here or in sub-headings that which part of the code is covered with mocha and which is covered with jest. And how we are merging the coverage of both to show in the repository tags.

Reply:
@nazarhussain I added the description. Merging coverage results will be handled in this issue: https://github.com/LiskHQ/lisk/issues/2946